app/models/repository.rb

```diff
@@ -6,14 +8,16 @@ class Repository < ApplicationRecord
 
   attribute :discarded_by_id, :integer
 
-  belongs_to :team, optional: true
+  belongs_to :team
```

Comment:
What are we going to do with currently shared repos? There will be failed validations there, we need to think about it.

Reply:
It's ok